packages/core/content-manager/admin/src/pages/EditView/components/DocumentActions.tsx

```diff
@@ -768,113 +768,166 @@ const UpdateAction: DocumentActionComponent = ({
+            collectionType,
+            model,
+            documentId,
+            params,
+          },
+          transformData(document)
+        );
 
-          if (
-            'error' in res &&
-            isBaseQueryError(res.error) &&
-            res.error.name === 'ValidationError'
-          ) {
-            setErrors(formatValidationErrors(res.error));
-          } else {
-            resetForm();
-          }
+        if ('error' in res && isBaseQueryError(res.error) && res.error.name === 'ValidationError') {
+          setErrors(formatValidationErrors(res.error));
         } else {
-          const res = await create(
+          resetForm();
+        }
+      } else {
+        const res = await create(
+          {
+            model,
+            params,
+          },
+          transformData(document)
+        );
+
+        if ('data' in res && collectionType !== SINGLE_TYPES) {
+          navigate(
             {
-              model,
-              params,
+              pathname: `../${res.data.documentId}`,
+              search: rawQuery,
             },
-            transformData(document)
+            { replace: true, relative: 'path' }
           );
-
-          if ('data' in res && collectionType !== SINGLE_TYPES) {
-            navigate(
-              {
-                pathname: `../${res.data.documentId}`,
-                search: rawQuery,
-              },
-              { replace: true, relative: 'path' }
-            );
-          } else if (
-            'error' in res &&
-            isBaseQueryError(res.error) &&
-            res.error.name === 'ValidationError'
-          ) {
-            setErrors(formatValidationErrors(res.error));
-          }
+        } else if (
+          'error' in res &&
+          isBaseQueryError(res.error) &&
+          res.error.name === 'ValidationError'
+        ) {
+          setErrors(formatValidationErrors(res.error));
         }
-      } finally {
-        setSubmitting(false);
       }
-    },
+    } finally {
+      setSubmitting(false);
+    }
+  }, [
+    clone,
+    cloneMatch?.params.origin,
+    collectionType,
+    create,
+    document,
+    documentId,
+    formatMessage,
+    formatValidationErrors,
+    isCloning,
+    model,
+    modified,
+    navigate,
+    params,
+    rawQuery,
+    resetForm,
+    setErrors,
+    setSubmitting,
+    toggleNotification,
+    update,
+    validate,
+  ]);
+
+  // Auto-save on CMD+S or CMD+Enter on macOS, and CTRL+S or CTRL+Enter on Windows/Linux
+  React.useEffect(() => {
+    const keysToListenTo = ['Enter'];
+
+    /**
+     * Detect the Arc browser specifically, because it registers CMD-S/CTRL-S for the quite common
+     * toggle sidebar action. So ignore that shortcut for it to avoid conflicts.
+     */
+    const isArcBrowser =
+      getComputedStyle(window.document.documentElement).getPropertyValue('--arc-palette-title')
+        .length > 0;
+
+    if (!isArcBrowser) {
+      keysToListenTo.push('s');
+    }
```

Comment:
I can get on board with merging this..._but_, what if we just don't include the ctrl+s / cmd+s part for now and do it in a follow up _if_ that's what people want.

We can maintain one command for now that works for all browsers and all flavors of browsers. My original comment about cmd+s was just that it was the most intuitive thing. I think if users learn what cmd+enter does they will be happy with that.

Adding cmd+s later is easy. If we add it now, we are stuck with maintaining both commands for the same action until a major release.

I'm particularly not crazy about the special treatment of the hype train that is Arc.

Reply:
Let me check with Yannis 😁 As I said on Slack, I'm not a fan of cmd+s on the web, so I'm just trying to please everyone but I'd be happy to remove it. I'm not sure we need to worry about backward compatibility for this though

Reply:
>I'm not sure we need to worry about backward compatibility

Reply:
> trying to please everyone

Nahhhh let's do what's best for the codebase and the product 🎈

Reply:
From a product perspective, it makes sense to start with Cmd+Enter as the primary save shortcut. It’s straightforward, works across all browsers, and avoids potential conflicts. We can gather user feedback after release and, if there’s significant demand, add Cmd+S later in a follow-up.